### PR TITLE
do not save token Id, if value is null

### PR DESCRIPTION
**Pull request by bratelefant**
## Summary
Do not save token Id on logging in, if no value is provided

## Linked issue(s)
#148 

## Involved parts of the project
`User.js`

## Reproduction
cf. the PR; however, I could observe improved resuming of a suspended iOS app, successfully restoring the session user.

### CODE REVIEW

**Comment by bratelefant** [comment]
@jankapunkt Can you check why lint fails? Local `npm run lint` yields no errors, after I did a `npm run lint:fix`.

File: src/Tracker.js [hunk 1]
```diff
@@ -57,10 +57,10 @@ function _debugFunc() {
...
     : typeof console !== 'undefined' && console.error
-    ? function () {
-        console.error.apply(console, arguments);
-      }
-    : function () {};
+      ? function () {
+          console.error.apply(console, arguments);
+        }
+      : function () {};
 }
```
Comment (jankapunkt) [inline src/Tracker.js:64]: I think the linter has some weird understanding of indentation on nested ternary operators. Maybe this one might work:

```js
if (typeof Meteor !== 'undefined') {
  return Meteor._debug;
}

return typeof console !== 'undefined' && console.error
  ? function () { console.error.apply(console, arguments); }
  : function () {};
```

File: src/user/User.js [hunk 2]
```diff
@@ -182,7 +182,15 @@ const User = {
   _isTokenLogin: false,
   _isCallingLogin: false,
   _loginWithToken(value) {
-    Data._tokenIdSaved = value;
+    if (!value) {
+      Meteor.isVerbose &&
+        console.info(
+          'User._loginWithToken::: parameter value is null, do not save token.'
```
Comment (jankapunkt) [inline src/user/User.js:188]: Maybe do not say "do not save token" as it sounds imperative like "you shall not save token" :mage_man: which could be confusing

Maybe rather print "will not save as token"

**Comment by bratelefant** [comment]
Ok looks like lint is working now. Could figure out why it did not throw any errors on my box though...

**Review by jankapunkt — APPROVED** [review]

**Pull request merged**